extensions/ipynb/src/deserializers.ts

```diff
@@ -53,6 +53,14 @@ const orderOfMimeTypes = [
 	'text/plain'
 ];
 
+function isEmptyVendoredMimeType(outputItem: NotebookCellOutputItem) {
+	if (outputItem.mime.startsWith('application/vnd.')) {
+		try {
+			return Buffer.from(outputItem.data).toString().length === 0;
+		} catch { }
+	}
+	return false;
+}
 function sortOutputItemsBasedOnDisplayOrder(outputItems: NotebookCellOutputItem[]): NotebookCellOutputItem[] {
 	return outputItems.sort((outputItemA, outputItemB) => {
```

Comment:
Considering that checking the index of an output item is not trivial, doing a sort directly here will lead to running `orderOfMimeTypes.findIndex` multiple times.

Maybe we can do `outputItems.map(item => ({ item, index: findIndexOfMimeType(item) }).sort((a, b) => a -b)`

Reply:
Yup missed that